datasets/coco2014/metadata.json

```diff
@@ -297,13 +297,11 @@
         },
         {
           "name": "bbox",
-          "@type": "ml:Field",
-          "description": "The bounding box around annotated object[s].",
+          "@type": "ml:BoundingBox",
```

Comment:
This should be ml:Field, right? BoundingBox is specificed in the dataType.

Reply:
done.

Reply:
done